spec/classes/database_postgresql_spec.rb

```diff
@@ -113,6 +113,53 @@
           it { is_expected.to contain_class('zabbix::params') }
         end
 
+        describe "when zabbix_type is server and version is #{zabbix_version} and manage_database_timescale is true" do
+          let :params do
+            {
+              database_name: 'zabbix-server',
+              database_user: 'zabbix-server',
+              database_password: 'zabbix-server',
+              database_host: 'node01.example.com',
+              zabbix_type: 'server',
+              zabbix_version: zabbix_version,
+              manage_database_timescale: true
+            }
+          end
+
+          it { is_expected.to compile.with_all_deps }
+          it { is_expected.to contain_exec('update_pgpass').with_command('echo node01.example.com:6432:zabbix-server:zabbix-server:zabbix-server >> /root/.pgpass') }
+          it { is_expected.to contain_exec('zabbix_server_create.sql').with_command("cd #{path} && if [ -f #{sql_server}.gz ]; then gunzip -f #{sql_server}.gz ; fi && psql -h 'node01.example.com' -U 'zabbix-server' -p 6432 -d 'zabbix-server' -f #{sql_server} && touch /etc/zabbix/.schema.done") }
```

Comment:
Typo

```suggestion
          it { is_expected.to contain_exec('zabbix_server_create.sql').with_command("cd #{path} && if [ -f #{sql_server}.gz ]; then gunzip -f #{sql_server}.gz ; fi && psql -h 'node01.example.com' -U 'zabbix-server' -p 5432 -d 'zabbix-server' -f #{sql_server} && touch /etc/zabbix/.schema.done") }
```

Reply:
Hey @bdeferme - just curious if you saw this.  Asking because I came here looking for the timescaledb support